core/src/main/java/org/arquillian/cube/impl/client/container/ContainerConfigurationController.java

```diff
@@ -64,15 +64,16 @@ public void remapContainer(@Observes BeforeSetup event, CubeRegistry cubeRegistr
         Object newConfigurationInstance = configurationClass.newInstance();
 
         for (PropertyDescriptor configurationClassPortField : configurationClassPortFields) {
+            // Move it outside if condition below, if there is a group of containers.
```

Comment:
What's the point of this comment here? Does it make sense to anyone reading the source code (and not this commit diff)?